src/pages/SignUp.js

```diff
@@ -1,12 +0,0 @@
-import React from 'react';
-
-import '../styling/SignIn.css';
-
```

Comment:
Just advice: If it is a personal project, completely empty files serve a purpose since you personally will know what is meant to go here. But if this was a professional project, it is best practice NOT to send up blank files like this INSTEAD just add some boiler plate as the Not Found looks